core/tests/wdio.conf.js

```diff
@@ -163,6 +191,9 @@ exports.config = {
   // specific capability.
   capabilities: [{
     browserName: 'chrome',
+    'goog:loggingPrefs': {
+      browser: 'ALL',
```

Comment:
What does this do?

Reply:
Generally `browser.getLogs('browser')` in webdriverio only returns the warnings and error present in console while in protractor it returns everything present in console. It is fine for every other suite as there we only need to checks the errors but in `embedding` suite we need to get all the text present inside console and to do that we need to add these lines in the wdio.conf.js file.